icechunk/src/lib.rs

```diff
@@ -46,3 +47,23 @@ mod private {
     /// See https://rust-lang.github.io/api-guidelines/future-proofing.html#sealed-traits-protect-against-downstream-implementations-c-sealed
     pub trait Sealed {}
 }
+
+#[cfg(feature = "logs")]
+pub fn initialize_tracing() {
+    use tracing_error::ErrorLayer;
+    use tracing_subscriber::{
+        layer::SubscriberExt, util::SubscriberInitExt, EnvFilter, Layer, Registry,
+    };
+
```

Comment:
We have two `Layers`. One keeps track of the spans to feed the `ICError` instances. The other is the one spitting logs to stdout. Filtering only applies to the second one.

Reply:
Can you convert that to a comment in the code please?

Reply:
done